packages/pn-pa-webapp/src/pages/NotificationDetail.page.tsx

```diff
@@ -314,6 +314,22 @@ const NotificationDetail: React.FC = () => {
                     downloadFilesMessage={getDownloadFilesMessage('attachments')}
                     downloadFilesLink={t('detail.download-files-link', { ns: 'notifiche' })}
                   />
+                  {notification.radd && notification.recipients.length === 1 && (
```

Comment:
@SarahDonvito since the first part of this condition is the same in both scenarios (mono/multi) I suggest to isolate the behaviour linked to the second part of this condition 
eg
`
{notification.radd && <Alert severity={'success'} sx={{ mb: 3, mt: 2 }} data-testid="raddAlert"> .... 
if notification.recipients.length === 1 && < ... mono message> 
else if notification.recipients.lenght > 1 && <multi message> ....
`